### ISSUE TITLE
How to deal with slide links (.pdf) on pages in the Script namespace?

### ISSUE THREAD

**Issue by rlaemmel** [101companies/101wiki#111]
I guess we need to discuss this.

Have a look here:

http://101companies.org/wiki/Script:The%20Expression%20Problem

I am putting the slides link into an extra section.

That's probably NOT the right way to do.

We may want to use the linksTo property.

a) We can't do this currently because of issue #107.
b) Even if that issue is resolved, I am not sure how to exactly to use it for this purpose.

If we were doing this:

<pre>
[[linksTo::http://userpages.uni-koblenz.de/~laemmel/pttcourse/slides/xproblem.pdf]]
</pre>

... then it will not be discoverable enough. (People wouldn't see that these are "the" slides.)

So perhaps we should be doing this:

<pre>
[[linksTo::http://userpages.uni-koblenz.de/~laemmel/pttcourse/slides/xproblem.pdf | Slides (.pdf)]]
</pre>

i) Would that work?
ii) How could we make sure that all pages agree on the markup?

Let's discuss please.

**Comment by tschmorleiz** [comment]
I think currently all "linksTo" triples are put into the "Resources" section. 

We could in fact have a special convention: When the link is named (here "Slides") the triples are put into a new section with that exact name ("Slides"). That can be used for all kinds of triples with all kinds of predicates.

**Comment by avaranovich** [comment]
I like the idea of

```
[[linksTo::http://userpages.uni-koblenz.de/~laemmel/pttcourse/slides/xproblem.pdf | Slides (.pdf)]]
```

This is absolutely valid mediawiki markup and rendered as expected. @tschmorleiz what's the point of inventing any conventions?

**Comment by avaranovich** [comment]
related discussion https://github.com/101companies/101wiki/issues/150

**Comment by burgua** [comment]
implemented in #150

**Comment by rlaemmel** [comment]
Cute! :-)
